tests/interception

```diff
@@ -112,6 +112,14 @@ if hasNeededAPIExtension container_syscall_intercept_finit_module; then
     # upload module file into the container
     lxc file push "${MODULE_PATH}" "c1/root/"
 
+    # deal with zstd compressed .ko
```

Comment:
Could you expand comment here to explain ehat "deal with" means in this context and why its needed.